src/app/pages/shell/shell.component.ts

```diff
@@ -88,11 +88,16 @@ export class ShellComponent implements OnInit, OnChanges, OnDestroy {
   onRightClick(): false {
     this._snackbar.open(
       T(
-        'Context menu copy and pasting operations \
-        are disabled in the Shell. If the expected \
-        keyboard shortcuts are not working, try \
-        Ctrl + Insert to copy or Shift + Insert to \
-        paste.'
+        'Context menu Copy and Paste operations \
+        are disabled in the Shell. Try using the \
+        default keyboard shortcuts to copy and \
+        paste. Copy shortcuts are <b>Ctrl+c</b> \
+        for Windows and <b>Command+c</b> for Mac. \
+        Pate shortcuts are <b>Ctrl+v</b> for \
+        Windows and <b>Command+v</b> for Mac. \
+        If the default keyboard shortcuts do not \
+        work, try <b>Ctrl+Insert</b> to copy and \
+        <b>Shift+Insert</b> to paste.'
```

Comment:
From our testing the Windows shortcuts are also Ctrl+Insert and Shift+Insert, Mac is the only one that lets you do Command+c and Command+v

Reply:
So it's Ctrl+c and Ctrl+v do not work at all in Windows?

Reply:
I believe only Ctrl+Insert and Shift+Insert, I do not have a windows box myself to test this with though, @aervin was the one who did the testing

Reply:
I suspect the reason they do this is because you would normally want "Ctrl+C" to be able to terminate something from the shell

Reply:
Windows only worked with the Insert combo @a-stjohn , at least in my testing.

Reply:
Excellent. Thanks for the info. Updating PR with the feedback.